charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -105,19 +105,19 @@ Get otel memory_limiter ballast_size_mib value based on 40% of resources.memory.
 Default config override for agent collector deamonset
 */}}
 {{- define "opentelemetry-collector.agentConfigOverride" -}}
+{{- if .Values.standaloneCollector.enabled }}
 exporters:
-  {{- if .Values.standaloneCollector.enabled }}
   otlp:
     endpoint: {{ include "opentelemetry-collector.fullname" . }}:55680
     insecure: true
-  {{- end }}
+{{- end }}
 
+{{- if .Values.standaloneCollector.enabled }}
```

Comment:
Similarly, `service` is a mandatory config section.

Reply:
as above, except for `service.piplines` instead of `exporters`